dividend_collection.py

```diff
@@ -0,0 +1,29 @@
+from dividend_payment import DividendTransaction
+
+
+class DividendCollection:
+    def __init__(self):
+        self.payments = []
+
+    def append(self, dividend_payment: DividendTransaction):
+        self.payments.append(dividend_payment)
+
+    def is_empty(self):
+        return not self.payments
+
+    def get_security(self):#get_fund_names
+        fund_names = []
+        for payment in self.payments:
+            if not payment.fund_name in fund_names:
+                fund_names.append(payment.fund_name)
+        return fund_names
+
+    def get_fund_payments(self, fund_name):
```

Comment:
```suggestion
    # Returns all dividend payments for the given fund.
    def get_fund_payments(self, fund_name):
```

I would add a line of documentation here, because I was confused for 1 second what this was doing.